cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -307,6 +306,37 @@ CAST_NUMERIC_FROM_STRING(double, arrow::DoubleType, FLOAT8)
 
 #undef CAST_NUMERIC_FROM_STRING
 
+#define CAST_NUMERIC_FROM_VARBINARY(OUT_TYPE, ARROW_TYPE, TYPE_NAME)                   \
+  GANDIVA_EXPORT                                                                       \
+  OUT_TYPE gdv_fn_cast##TYPE_NAME##_varbinary(int64_t context, const char* data,       \
```

Comment:
Is it any different from CAST_NUMERIC_FROM_STRING functions. Can't you just call them

Reply:
Use a macro to use the same definition for both string and varbinary inputs

Reply:
done